rclpy/rclpy/qos.py

```diff
@@ -270,6 +270,9 @@ class QoSHistoryPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_HISTORY_KEEP_ALL = 2
     KEEP_ALL = RMW_QOS_POLICY_HISTORY_KEEP_ALL
 
+#Alias with the old name, for retrocompatibility
+QoSHistoryPolicy = HistoryPolicy 
+
 
 class QoSReliabilityPolicy(QoSPolicyEnum):
```

Comment:
You didn't renamed the classes, this should be `ReliabilityPolicy`.
Same for the other three cases (L259, L295, L313).

Reply:
Done..